pmm/pmm2-release.groovy

```diff
@@ -520,26 +518,23 @@ ENDSSH
         stage('Set git release tags') {
             steps {
                 deleteDir()
-                unstash 'copy-stash'
                 withCredentials([sshUserPrivateKey(credentialsId: 'GitHub SSH Key', keyFileVariable: 'SSHKEY', passphraseVariable: '', usernameVariable: '')]) {
                     sh '''
-                        set -x
-                        # Do not allow this step to fail so we can create tags outside of the pipeline
+                        # This step should never cause the pipeline failure so we can create tags outside of the pipeline
                         set +e
-                        cat copy.list
+                        set -x
 
                         # List of repos whose release branches need to be tagged
-                        # TODO: add pmm-submodules to the list, maybe even fallback to using submodules
                         declare -A repos=(
                             ["percona-grafana"]="percona-platform/grafana"
                             ["percona-dashboards"]="percona/grafana-dashboards"
-                            ["pmm-update"]="percona/pmm-update"
                             ["pmm"]="percona/pmm"
+                            ["pmm-submodules"]="Percona-Lab/pmm-submodules"
                         )
 
                         # Configure git settings globally
                         git config --global advice.detachedHead false
-                        git config --global user.email "dev-services@percona.com"
+                        git config --global user.email "noreply@percona.com"
```

Comment:
why we changed email?

Reply:
To prevent spammers from sending emails to percona. I think it's part of best practices to not disclose corporate email boxes in OSS code. I'm trying to make code better :)

Reply:
That said, I can revert this change, no big deal.